test/__fixtures/BaseTest.sol

```diff
@@ -27,6 +27,7 @@ abstract contract BaseTest is Test {
     constructor()
     {
         // set shared variables
+        _factory.set(keccak256("ConstantProductPair::platformFeeTo"), bytes32(uint256(uint160(_platformFeeTo))));
         _factory.set(keccak256("ConstantProductPair::swapFee"), bytes32(uint256(30)));
         _factory.set(keccak256("ConstantProductPair::platformFee"), bytes32(uint256(2500)));
         _factory.set(keccak256("ConstantProductPair::defaultRecoverer"), bytes32(uint256(uint160(_recoverer))));
```

Comment:
Thoughts on refactoring this to use library? Would be good to lock in/test the library approach

Reply:
actually you didn't have to review this PR cuz it was the other one that mattered. I was going to close this PR. 

By library you mean like managing the setting + casting of these variables?

Reply:
yes

Reply:
will fix in a separate PR as this PR is old and no longer relevant